agent/src/beerocks/fronthaul_manager/monitor/monitor_thread.cpp

```diff
@@ -64,9 +64,13 @@ monitor_thread::monitor_thread(const std::string &slave_uds_, const std::string
 
     using namespace std::placeholders; // for `_1`
 
+    bwl::hal_conf_t hal_conf;
+    // FIXME: got ctrl iface path from [agent0] instead of section according "monitor_iface_"
+    hal_conf.wpa_ctrl_path = beerocks_slave_conf.wpa_supplicant_ctrl_iface[0];
```

Comment:
It's needed to take not wpa supplicant ctrl path but hostapd ctrl path

Reply:
fixed